drake/multibody/rigid_body_plant/visualization/show_time.py

```diff
@@ -0,0 +1,26 @@
+# Note that this script runs in the main context of drake-visulizer,
+# where many modules and variables already exist in the global scope.
+from director import lcmUtils
+import bot_core as lcmbotcore
+import time
+
+def onViewerDraw(msg, last_time = [0, 0]):
+    t = msg.timestamp * 1e-3 # convert from milliseconds
+
+    rt0 = last_time[0]
+    rt1 = time.time()
+
+    t0 = last_time[1]
+
+    dt = t - t0
+    drt = rt1 - rt0
+
+    rt_ratio = dt / drt
+
+    last_time[0] = rt1
+    last_time[1] = t
+
+    my_text = 'sim time: %.3f, real time factor: %.3f' % (t, rt_ratio)
+    vis.updateText(my_text, '')
+
+lcmUtils.addSubscriber('DRAKE_VIEWER_DRAW', lcmbotcore.viewer_draw_t, onViewerDraw)
```

Comment:
efficiency improvement: add a hook into the drakevisualizer python code so that you can do this update without adding a new subscriber which requires its own decode (in python).  The drakevisualizer app already has a subscriber that decodes this message, but it doesn't store the message or provide a callback, so this would require some new design, but it would be a welcome improvement if designed properly.